src/chrome/content/rules/pronoun.is.xml

```diff
@@ -0,0 +1,5 @@
+<ruleset name="pronoun.is">
+	<target host="pronoun.is" />
```

Comment:
```
www.pronoun.is
hubots.pronoun.is
my.pronoun.is
```

Reply:
I don’t think these domain are available in HTTPS.

Reply:
@franklinyu Then add information about that as a comment.